src/plugins/autonomy/ExternalControl/ExternalControl.cpp

```diff
@@ -61,6 +65,28 @@ ExternalControl::ExternalControl() :
 void ExternalControl::init(std::map<std::string, std::string> &params) {
     server_address_ = get("server_address", params, std::string("localhost:50051"));
     delayed_task_->delay = get("timestep", params, 0.0);
+    num_attempts_ = get("num_attempts", params, 5);
+    use_trained_model_ = get("use_trained_model", params, false);
+    if (use_trained_model_) {
+        std::string xml_file = params.at("XML_FILENAME");
+        std::string actor = params.at("actor_name");
+
+        std::vector<std::string> address_parts;
+        boost::split(address_parts, params.at("server_address"), [](char c){return c == ':';});
+        std::string port_num = address_parts[1];
+        python_cmd_ = "external_control.py";
+        python_cmd_ += " --port-num " + port_num;
+        python_cmd_ += " --actor " + actor;
+        python_cmd_ += " --xml " + xml_file;
+        python_cmd_ += " &";
+        int result = system(python_cmd_.c_str());
```

Comment:
Can we just make this a single assignment? e.g., 
``python_cmd_ = "external_control.py"``
``    + " --port-num " + port_num``
``    + " --actor " + actor``
``    + " --xml" + xml_file``
``    + " &";``